frontend/packages/dev-console/locales/en/devconsole.json

```diff
@@ -362,12 +362,12 @@
   "Role": "Role",
   "Select a role": "Select a role",
   "Reload": "Reload",
-  "<0>{children} or <3>create a Project.</3></0>": "<0>{children} or <3>create a Project.</3></0>",
+  "<0>{{children}} or <3>create a Project.</3></0>": "<0>{{children}} or <3>create a Project.</3></0>",
```

Comment:
Hi, this is better, but we can't plugin any text as `children` here. Translators often have to move words around in a sentence depending on the language and need the full string as one message. I know it means some rework for `CreateProjectListPage`.

Reply:
Hi Sam , i looked into it and based on my understanding it will need refactoring in multiple components (around 10) where `CreateProjectListPage` is called from and problem with that is `create a Project` is button text which has action associated so it'll be duplicity in order to have whole sentence together.

another option is just moving `or` to the caller and have `button` and action in `CreateProjectListPage` but even then sentence will be incomplete.  i.e 

```
<CreateProjectListPage title={t('devconsole~Add')}>
     {t('devconsole~Select a Project to start adding to it or')}
 </CreateProjectListPage>
```

and `CreateProjectListPage` list as 

```
<ProjectListPage {...props} title={title}>
        {{ children }}
        <Button isInline variant="link" onClick={openProjectModal}>
          {t('devconsole~create a Project.')}
        </Button>
 </ProjectListPage>
```

Let me know your thoughts